The job **`mingw-check`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/9027116552/job/24805489926) [(plain)](https://github.com/rust-lang/rust/commit/336dd16d05f34d12763e3db296a9ae438719a3d9/checks/24805489926/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
   |
note: while trying to match `,`
  --> /checkout/library/core/src/macros/mod.rs:37:16
   |
37 |     ($left:expr, $right:expr $(,)?) => {

error: aborting due to 1 previous error

Couldn't compile the test.
Couldn't compile the test.

failures:
    library/core/src/result.rs - result (line 47)

test result: FAILED. 4627 passed; 1 failed; 42 ignored; 0 measured; 0 filtered out; finished in 145.14s

error: doctest failed, to rerun pass `-p core --doc`
  local time: Fri May 10 04:27:31 UTC 2024
  network time: Fri, 10 May 2024 04:27:31 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/library/core/src/result.rs
+++ b/library/core/src/result.rs
@@ -51,7 +51,7 @@
 //! assert!(good_result.is_ok() && !good_result.is_err());
 //! assert!(bad_result.is_err() && !bad_result.is_ok());
 //!
-//! // `map` and `map_err` consume the `Result` and produces another.
+//! // `map` and `map_err` consume the `Result` and produce another.
 //! let good_result: Result<i32, i32> = good_result.map(|i| i + 1);
 //! let bad_result: Result<i32, i32> = bad_result.map_err(|i| i - 1);
 //! assert_eq!(good_result, Ok(11));

--- a/library/core/src/result.rs
+++ b/library/core/src/result.rs
@@ -67,7 +67,7 @@
 //!
 //! // Consume the result and return the contents with `unwrap`.
 //! let final_awesome_result = good_result.unwrap();
-//! assert_eq!(final_awesome_result)
+//! assert!(final_awesome_result)
 //! ```
 //!
 //! # Results must be used

